src/lib/utilities/history-summary-plucker.ts

```diff
@@ -0,0 +1,93 @@
+import { findValueIfItIncludesWord } from './find-value-if-it-includes-word';
+
+const summaryExtractors = {
+  ActivityTaskCancelRequested: (d) => ({ Id: d.activityId }),
```

Comment:
I'm not really sure what `d` is in all of these. Can we use a more descriptive name?

Reply:
This code is also super unsafe from a type perspective. All of this would benefit greatly from typing the incoming value 

```typescript
  ActivityTaskCancelRequested: (d:WhateverNetworkValueThisRepresents) => ({ Id: d.activityId }),
```